### ISSUE TITLE
`isInAws` always returns `false`

### ISSUE THREAD

**Issue by meirna** [astuyve/lambda-stream#6]
Hey guys, thanks for this library!

I'm having an issue with [`isInAws` function](https://github.com/astuyve/lambda-stream/blob/76fe5364758b1c87d22e461ccdfc9463b925e6e6/src/index.ts#L9-L14): it always returns `false`, even in AWS Lambda. 

It works for me if I change the condition to:

```
function isInAWS() {
    return typeof awslambda !== 'undefined';
}
```

Do you know what might be the reason for this?

**Comment by bmul** [comment]
Same

**Comment by astuyve** [comment]
Hey folks, thanks for flagging this - I was traveling the last few weeks. I'll take a look soon.

I copied this from code we used in [datadog's instrumentation](https://github.com/DataDog/datadog-lambda-js/blob/main/src/index.ts#L129-L130) which as far as I know still works, however the runtime can always change on us.

Thanks!

**Comment by astuyve** [comment]
hey @meirna, @bmul - I've prepared a patch in #7 which seems to work in my manual testing. I'll cut a new release tomorrow.

Thanks!

**Comment by astuyve** [comment]
Published in [v0.4.0](https://github.com/astuyve/lambda-stream/releases/tag/0.4.0)
